iteration.Rmd

```diff
@@ -846,7 +846,8 @@ x %>%
 
 ```{r, eval = FALSE}
 library(ggplot2)
-plots <- mtcars %>% 
+plots <-
```

Comment:
Can you leave this as is for now?

Reply:
Sorry I missed this comment for so long...
Just reverted that change.